library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -571,6 +571,9 @@ private boolean doSelectDate(Date date, MonthCellDescriptor cell) {
     switch (selectionMode) {
       case RANGE:
         if (selectedCals.size() > 1) {
+          if(!isDateSelectable(date)){
+            return false;
+          }
```

Comment:
In here I'm trying to prevent creating a range that opens with a date that isn't selectable but was unblocked due to previously being the departure date